### ISSUE TITLE
Missing dictionaries folder in 2.17.7 on Windows 10

### ISSUE THREAD

**Issue by rgembalik** [RocketChat/Rocket.Chat.Electron#1534]
## My Setup

- Operating System: Windows 10 x64
- App Version: 2.17.6
- Installation type: exe

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
On Windows 10, I upgraded RC from 2.17.1 to 2.17.7 by manually downloading and launching the installer. After that I was missing pl_PL spellchecking option, so I downloaded some dictionaries from the Internet and tried to install them. Immediately I got an error about RC not being able to install the dictionary files.
![image](https://user-images.githubusercontent.com/2618837/75016683-d3dec200-548b-11ea-8c85-c146531b4358.png)

After messing around, digging here and there and trying launching as an Administrator (nothing helped) I noticed somewhere, that RC should load dictionaries from `C:\Users\USERNAME\AppData\Roaming\Rocket.Chat\dictionaries`, but I didn't have that directory. I created it and only after this I was able to install the dictionaries from `dic/aif` files.

### Current Behavior
Custom Spell checking dictionaries won't install until the dictionaries folder is created manually.

### Expected Behavior
The dictionaries folder should be created already after installation or should be automatically created when loading dictionaries.

**Comment by ankar84** [comment]
Yes, I described it in https://github.com/RocketChat/Rocket.Chat.Electron/issues/1497#issuecomment-586094019 and https://github.com/RocketChat/Rocket.Chat.Electron/issues/1520#issuecomment-586111288
By default dictionaries folder in path c:\Users\UserName\AppData\Roaming\Rocket.Chat\dictionaries do not exist, and it is can be a bug.

**Comment by AWestermeyer** [comment]
Hi !
Same here, but I'm on **macOS Catalina 10.15.3** with **Rocket.Chat 2.17.9**
Where should I put the dictionaries files on macOS ?

Is it in ~/Library/Containers/chat.rocket/Data/Library/Application Support/Rocket.Chat/ ?
But there is no "dictionaries" folder.

Thanks

**Comment by homberger** [comment]
I am/was still facing this problem of the missing folder `dictionaries` with Client version 1.17.11 on Windows 10.

Maybe this error prevented the loading of the Rocket.Chat Client sometimes and only a forced reload (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>R</kbd>) achieved a full loading of the Rocket.Chat UI. It could have been a coincidence, though.

This error was reported in dev tools:
```
app.js:6156 Error: ENOENT: no such file or directory, scandir 'C:\Users\MyUsername\AppData\Roaming\Rocket.Chat\dictionaries'
getPairsOfDictionaryFiles @ app.js:6156
```

After creating the missing directory the UI was loading without problems since then. But still: could be an coincidence.

**Comment by ankar84** [comment]
Fixed in 3.0
![image](https://user-images.githubusercontent.com/4023037/94756136-52df9580-039f-11eb-851f-3fcef7ae468f.png)

**Comment by jeanfbrito** [comment]
Please test if the latest release did solve this. If not please reopen the issue or create a new one and tell us. I will be closing this one for now.

Thank you!
